card/mod/05_standard/spec/set/right/account_spec.rb

```diff
@@ -145,11 +145,11 @@
     context 'denied access' do
       it 'excludes protected subcards' do
         skip
-        Card.create(:name=>"A+B+*self+*read", :type=>'Pointer', :content=>"[[u1]]")
-        u2 = Card.fetch 'u2+*following', :new=>{:type=>'Pointer'}
+        Card.create(name: "A+B+*self+*read", type: 'Pointer', content: "[[u1]]")
+        u2 = Card.fetch 'u2+*following', new: {type: 'Pointer'}
         u2.add_item "A"
         a = Card.fetch "A"
-        a.update_attributes( :content=> "new content", :subcards=>{'+B'=>{:content=>'hidden content'}})
+        a.update_attributes( content: "new content", subcards: {'+B'=>{content: 'hidden content'}})
```

Comment:
Line is too long. [99/80]<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.<br>Space inside parentheses detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.